src/main/java/com/compstore/entity/pc/PCEntity.java

```diff
@@ -30,10 +32,12 @@ public class PCEntity {
     private String graphicsCardName;
 
     @Column(name = "ram_gb_capacity")
-    private Integer ramGBCapacity;
+    @Enumerated(value = EnumType.STRING)
```

Comment:
Zastanawiam się, czy teraz odkąd jest to string to czy nie lepiej usunąć GB z nazwy pola i dawać np. `ramCapacity`. Wtedy będzie w filtach mogłobyć np. 512GB oraz 1TB zamiast 1024GB. Użytkownik już nie będzie podawał nic z palca więc nie trzeba trzymać się już jednej jednostki

Reply:
Tak, zmienimy nazwę pola: 
- `ramGBCapacity` na `ramCapacity`
- `driveGBCapacity` na `driveCapacity`